resources/styles/components/learning-guide.less

```diff
@@ -245,3 +249,151 @@
     }
   }
 }
+
+
+
+// IE10/11 resize hack
+// these breakpoints correspond approx. with what height:auto calculates in other browsers.
+// only targets IE10 and above.
+
+@media screen and (max-width: 1079px) {
```

Comment:
Can you combine these into a single mixin? That way the breakpoints are each one line